katsdpimager/report.py

```diff
@@ -102,6 +102,8 @@ def __init__(self, dataset: katdal.DataSet,
         self.peak: u.Quantity = [math.nan] * common.channels * (u.Jy / u.beam)
         # Noise per channel (NaN where missing)
         self.noise: u.Quantity = [math.nan] * common.channels * (u.Jy / u.beam)
+        # Estimate neight from weights (NaN where missing)
```

Comment:
Do you mean 'noise from weights'?

Reply:
Indeed. Sometimes I wonder about my brain.